package.json

```diff
@@ -124,7 +125,7 @@
     "truffle-hdwallet-provider": "1.0.17",
     "ts-node": "8.6.2",
     "typechain": "1.0.5",
-    "typechain-target-trufflehotfix": "0.0.2-alpha",
+    "typechain-target-trufflehotfix": "0.0.4-alpha",
```

Comment:
what exactly is this? no readme explains. it looks (from title) like a generic fix, but it depends on `openeth/truffle-typings` - so its NOT generic..

Reply:
It is not generic fix, it fixed a few typing issues I had because TypeChain uses truffle 4 and we use truffle 5. 
I cannot publish this package under 'openeth' because the lookup for it is hardcoded in TypeChain itself